mobile/src/main/java/ch/epfl/sdp/mobile/infrastructure/speech/android/AndroidSpeechRecognizerFactory.kt

```diff
@@ -0,0 +1,72 @@
+package ch.epfl.sdp.mobile.infrastructure.speech.android
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.speech.RecognizerIntent.*
+import android.speech.SpeechRecognizer as NativeSpeechRecognizer
+import android.speech.SpeechRecognizer.RESULTS_RECOGNITION
+import ch.epfl.sdp.mobile.infrastructure.speech.SpeechRecognizer
+import ch.epfl.sdp.mobile.infrastructure.speech.SpeechRecognizerFactory
+
+/** The default locale we'll be using for speech recognition. */
+private const val DefaultLanguage = "en-US"
+
+/** The default maximum number of results we'll receive from the system speech recognizer. */
+private const val DefaultResultsCount = 10
+
+/**
+ * An implementation of a [SpeechRecognizer] which is backed by a [NativeSpeechRecognizer].
```

Comment:
The documentation is the same that `AndroidSpeechRecognizer`, maybe you can add more information to distinguish them?

Reply:
Indeed, it shouldn't be. I've fixed it in 288f267d8e281a4f4bc2bdb6c8accc6233e1d760.